### PR TITLE
Basic site

### PR DESCRIPTION
**Pull request by rpwagner**
This is an initial draft of a site for the CMB-S4 data repository proof-of-concept. It couples a catalog of data releases and datasets hosted on GitHub Pages with a collection of the files in a Globus collection. Data access to the files are managed via permissions on the collection, including Globus Groups that users can self-register to join, or request approval. Structured metadata for the datasets is provided using Schema.org JSON-LD through YAML headers on the dataset Markdown pages. Datasets have a JSON manifest file including the URL, SHA512 checksum, and size in bytes of the files for validation.

Potential additions:

- Add more datasets from the data provided.
- Defining the structure of data releases, versions, and datasets.
- Page styling via templates.
- Additional metadata for the data release and datasets.
- Documentation on how to access the dataset files using HTTPS and Globus Transfer.
- Creation of a data stewards group to manage permissions and review data access requests.
- Examples of how to retrieve data using the BDBag CLI.

### CODE REVIEW

File: makefiletable.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+# Takes a remote file manifest JSON file and creates a Markdown table
+# https://github.com/fair-research/bdbag/blob/master/doc/config.md#remote-file-manifest
+
```
Comment (zonca) [inline makefiletable.py:5]: @rpwagner you run these 2 scripts manually on your machine to generate the markdown files, right?

Comment (zonca) [reply makefiletable.py:5]: but the output markdown files are on Globus, not on Github

File: releases/npipe6v20/fullsky.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+This dataset is available via Globus Transfer or HTTPS. [Click here](https://app.globus.org/file-manager?origin_id=38f01147-f09e-483d-a552-3866669a846d&origin_path=%2Fdatareleases%2Fnpipe6v20%2Ffullsky%2F) to view the files in the Globus web app.
+
+## Files
+
+- Number of files: 27
+- Total size: 10.4 GiB
+- [JSON format file manifest](https://g-456d30.0ed28.75bc.data.globus.org/datareleases/npipe6v20/fullsky/manifest.json)
+
+|                                                                    File Name                                                                    |   Size    |     Hash     |
+| ----------------------------------------------------------------------------------------------------------------------------------------------- | --------- | ------------ |
```
Comment (zonca) [inline releases/npipe6v20/fullsky.md:34]: @rpwagner how do you generate this list of files?

File: makefiletable.py (hunk 1 above)
Comment (rpwagner) [reply makefiletable.py:5]: I'll write up a Markdown file describing the process. It was somewhat messy and ad hoc for these first datasets.

**Pull request merged**